stories/plotxy.stories.tsx

```diff
@@ -0,0 +1,147 @@
+import { ComponentStory, Meta } from '@storybook/react';
+import React from 'react';
+
+import data from '../src/data/variables.json';
+import PlotXY, {
+  Data,
+  DataXML,
+  Margins,
+  XmlPlotXY,
+} from '../src/layout/PlotXY';
+import LineSerie from '../src/layout/PlotXY/components/LineSerie';
+import XYAxis from '../src/layout/PlotXY/components/axis/xy-axis';
+
+export default {
+  title: 'Layout/PlotXY',
+  args: {},
+} as Meta;
+interface PlotMultiLinesStoryProps {
+  dataArray: Data[][];
+  margins?: Margins;
+  height: number;
+  width: number;
+  ticks?: number;
+  xLabel?: string;
+  yLabel?: string;
+}
+const MultiLinesStory: ComponentStory<
+  (props: PlotMultiLinesStoryProps) => JSX.Element
+> = (props: PlotMultiLinesStoryProps) => {
+  const {
+    dataArray = [],
+    margins,
+    height,
+    width,
+    ticks,
+    xLabel = 'x',
+    yLabel = 'y',
+  } = props;
+  return (
+    <PlotXY
+      dataArray={dataArray}
+      margins={margins}
+      width={width}
+      height={height}
+      ticks={ticks}
+    >
+      <XYAxis
+        xLabel={xLabel}
+        yLabel={yLabel}
+        labelStyle={{ fontSize: '18px' }}
+      />
+      {dataArray.map((data, key) => {
+        return (
+          <LineSerie
+            // eslint-disable-next-line react/no-array-index-key
+            key={key}
+            data={data}
+            color={`#${Math.floor(Math.random() * 16777215).toString(16)}`} //random colors
+          />
+        );
+      })}
+    </PlotXY>
+  );
+};
+export const MultiLines = MultiLinesStory.bind({});
+MultiLines.args = {
```

Comment:
can you add `ticks` to the controllable props?

Reply:
done